manifests/f/FastStone/Viewer/7.6/FastStone.Viewer.locale.en-US.yaml

```
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.1.0 $debug=AUSU.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: FastStone.Viewer
+PackageVersion: "7.6"
+PackageLocale: en-US
+Publisher: FastStone Soft
```

```suggestion
Publisher: FastStone Corporation
```
This must match Control Panel.